metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi-lexical-grammar.rule

```diff
@@ -331,7 +331,7 @@ parenthesized_expression:   "(" WS_INLINE? expression WS_INLINE? ")"
 
 not_implemented_expression: "..."
 
-invoke_expression:  "#"? primary_expression "(" NEWLINE? argument_list? NEWLINE? ")"
+invoke_expression:  "#"? primary_expression "(" NEWLINE? keyword? argument_list? NEWLINE? ")"
```

Comment:
please add comments to the top, in the MODIFICATIONS section

Reply:
done